source/ui/BioXAS/BioXASMainXASScanConfigurationView.cpp

```diff
@@ -232,3 +241,14 @@ void BioXASMainXASScanConfigurationView::onEdgeChanged()
 	if (energy_->value() != configuration_->energy())
 		energy_->setValue(configuration_->energy());
 }
+
+void BioXASMainXASScanConfigurationView::onEstimatedTimeChanged()
+{
+    configuration_->blockSignals(true);
```

Comment:
Take a look at some of the other beamline versions.  If it's a consistent thing, keep it in.  I don't think it's required though.